src/main/kubectl.ts

```diff
@@ -89,16 +89,15 @@ export class Kubectl {
   public static readonly bundledKubectlVersion: string = bundledVersion;
   public static invalidBundle = false;
   private static bundledInstance: Kubectl;
+  private static readonly versionRegex = /^v?(\d+\.\d+)(.*)/;
```

Comment:
Just noticed this regex.. just wondering why aren't we using `semver` package for parsing? 🤔 @nevalla do you remember?

Reply:
I moved this from where we were doing it before since I wanted to reuse it (ended up not doing so). I would say that this probably predates `semver` being used. I can change it if you want.

Reply:
Change should probably be a separate PR (I was just wondering why we had this regex).

Reply:
Okay I can factor that out.